.github/workflows/main.yml

```diff
@@ -52,12 +52,11 @@ jobs:
           rustup target add wasm32-unknown-emscripten
       - name: Set up rust (mt)
         if: ${{ matrix.threading == 'mt' }}
-        run: |        
+        run: |
```

Comment:
Do you have a formatter for YAML in VS code? I noticed that I sometimes involuntarily add those white spaces when pasting.

I tried the RedHat one but it changes single quotes to double quotes which I don't always want.

Reply:
VS code default "remove whitespaces"